Xamarin.Forms.Platform.Android/Renderers/ShellBottomNavViewAppearanceTracker.cs

```diff
@@ -149,6 +144,6 @@ protected virtual void Dispose(bool disposing)
 			}
 		}
 
-#endregion IDisposable
+		#endregion IDisposable
```

Comment:
Can you remove this extra space?

Reply:
This extra space has been created by ".editorconfig" settings (using ctrl + k, d)..
So, shouldn't regions have this kind of indentation?

<p align="center">
	<kbd>
		<img src="https://user-images.githubusercontent.com/19656249/66710234-8a969d00-ed4a-11e9-8806-c09501fe1b6a.png" alt="visual studio identation visualization for this code" style="max-width:100%;"/>
	</kbd>
</p>

Reply:
Yeah, my mistake. I confused with the default #if, #region really has this space